### ISSUE TITLE
Accessibility bug - wizard page focus is not on page elements

### ISSUE THREAD

**Issue by bharathpalaksha** [microsoft/azuredatastudio#25445]
- Azure Data Studio Version: 1.47.0
- OS Version: windows 11, mac and linux
- Extension: Azure Cosmos DB Migration for MongoDB

This accessibility bug is on extension but we use components from Azure data studio, wizard page focus goes to next button by default if the page doesn't have any input elements. Focus should also be on the link component if it's present.

Steps to Reproduce:

[2837057_A11y_Azure Cosmos DB Migration for MongoDB_Summary_Six pop-up_Keyboard.webm](https://github.com/microsoft/azuredatastudio/assets/342603/6cac70ea-2eee-408c-bd5f-dd0ed996ff03)

Does this issue occur when all extensions are disabled?: Yes/No

**Comment by Samhitha66** [comment]
#A11yMAS; #SH_Azure Data Studio_Nov2023_Linux_Dec2023; #SQL Azure Data Studio;
